DiceKit/Die.swift

```diff
@@ -36,6 +36,13 @@ public struct Die: Equatable {
     
 }
 
+/**
+Global convenience function for making new Die objects
+*/
+public func d(sides: Int = 6) -> Die {
```

Comment:
Instead of defaulting this, let's just leverage whatever the default is for `Die`. That would mean creating two `d()`, one that takes sides and one that doesn't. Both just call `Die()` (one with sides and one without). That way we don't need to change it in both spots, and it also doesn't assume anything about `Die()`.